nix/modules/blocksense/module-opts/default.nix

```diff
@@ -19,6 +19,10 @@ in
     Enable the Blocksense sequencer and reporter node services.
   '');
 
+  kafka.enable = mkEnableOption (mdDoc ''
+    Enable Kafka.
+  '');
```

Comment:
Why is this inside the `blocksense` *module* attribute now? Not saying that it shouldn't, just asking why.

On a similar note, why is `blama` inside too? Since it (or they?) can be useful outside the whole of `blocksense`, we might leave it out (leaving the possibility of writing a test someday just for `blama` alone)